website/docs/guides/markdown-features/markdown-features-assets.mdx

```diff
@@ -84,6 +84,27 @@ If you are using [@docusaurus/plugin-ideal-image](../../api/plugins/plugin-ideal
 
 :::
 
+:::note SVGs in CommonJS require and Import statement
+
+To use SVGs with CommonJS require and Import statements you need to override the default `svg-loader` behavior by prepending `!!url-loader!` to the asset path. For example:
```

Comment:
```suggestion
To import SVGs as an asset URL instead of a [React component](#inline-svgs), you need to override the default `svg-loader` behavior by prepending `!!url-loader!` to the asset path. For example:
```

Might also be useful to mention `configureWebpack` if you want to override it globally?